src/types/public.ts

```diff
@@ -56,3 +56,51 @@ export interface WebPushOptions {
   serviceWorkerUrl?: string;
   scope?: string;
 }
+
+export enum ActionParameterType {
+    Int = 'int',
+    Integer = 'integer',
```

Comment:
Do we need both?

Reply:
I don't think so, seems like one is used only for legacy.